styles/index.css

```diff
@@ -0,0 +1,232 @@
+@import url('https://fonts.googleapis.com/css2?family=Fascinate&family=Questrial&display=swap');
+
+body {
+    background: linear-gradient(45deg,#fec5bb,#fcd5ce,#fae1dd,#f8edeb,#e8e8e4,#d8e2dc,#ece4db,
+    #ffe5d9,#ffd7ba,#fec89a);
+    font-family: 'Questrial', sans-serif;
+}
+
+main {
+    display: flex;
+    flex-flow: wrap;
+    background-color: #fae1dd;
+    border-width: 1px;
+    border-style: solid;
+    border-color: #38040e;
+    border-radius: 5vw;
+    padding: 2vw;
+    margin: 4vw;
+    
+}
+
+main button {
+    background-color: #fcd5ce;
+    border-left-color: #fec89a;
+    border-top-color: #ffd7ba;
+    border-radius: 10vw;
+}
+
+main:nth-of-type(3) {
+    flex: 1 20vw;
+    order: 3;
+}
+
+header {
+    display: flex;
+    flex-direction: column;
+    align-content: space-evenly;
+}
+
+header h2 {
+    color: #38040e;
+    font-style: italic;
+    padding-left: 10vw;
+    font-size: 5em;
+}
+
+header h1 {
+    color: #38040e;
+    font-family: 'Fascinate', cursive;
+    align-self: flex-end;
+    margin-right: 10vw;
+    margin-bottom: 0.25vh;
+    margin-top: 0.25vh;
+    background-color: #f8edeb;
+    border-width: 1px;
+    border-style: solid;
+    border-color: #38040e;
+    padding: 2vw;
+}
+
+#cityInput {
+    flex: 5 100%;
+    margin: 1vw;
+    display: flex;
+    justify-content: space-evenly;
+    padding:4vw;
+}
+#cityInput button {
+    width: 10vw;
+}
+
+#weatherGarden {
+    flex: 3 60%;
+    display: flex; 
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin: 1vw;
+    height: 100vh;
+}
+
+#sky, #landScape {
+    width: 54vw;
+    margin-right:1vw;
+    /* object-fit:contain; */
+}
+#sky {
+    /* flex: 1; */
+    height: 40vh;
+    align-self: flex-start;
+}
+.skyRainy {
+    background-image: url('../assets/rain.png');
+    background-size: contain;
+}
+
+
+#landScape {
+    /* flex: 1; */
+    height: 27vh;
+    align-self: flex-end;
+    padding-top: 2vh;
+}
+.landHot {
+    background-image: url('../assets/hot.png');
+    background-size: contain;
+}
+.landTemperate {
+    background-image: url('../assets/comfy.png');
+    background-size: contain;
+}
+.landCold {
+    background-image: url('../assets/cold.png');
+    background-size: contain;
+}
+.landFreezing {
+    background-image: url('../assets/freezing.png');
+    background-size: contain;
+}
+
+main div {
+    flex: 2 30%;
+    margin: 1vw;
+    /* padding-bottom: 1vh; */
```

Comment:
There's a couple comments & some spacing between rule sets that seems inconsistent to me across this file, something to take a look at as a we clean up/refactor code.